### ISSUE TITLE
Move local repo to gannet

### ISSUE THREAD

**Issue by yaaminiv** [RobertsLab/resources#422]
In [this issue](https://github.com/RobertsLab/resources/issues/414) @kubu4 pointed out that hummingbird was out of storage. He suggested I move my local [project-virginica-oa](https://github.com/RobertsLab/project-virginica-oa) to gannet, and work from my repo there.

I could easily clone my repo from the web onto gannet, but what about all of the large files from my local repo that are in my `.gitignore`? Is there an easy way to move those over besides just copying over files listed in my `.gitignore`?

**Comment by sr320** [comment]
I would use rsync..

**Comment by sr320** [comment]
Though maybe we should chat about why you might have a repo that is so large.

**Comment by sr320** [comment]
That should be done on Mox

**Comment by yaaminiv** [comment]
Yes, I will run `bismark` on Mox in the future. However, I'd like to use RStudio on hummingbird to test different `methylKit` parameters while `bismark` is running on Mox. Seeing how there is limited storage on hummingbird because of all of the large files in my repo, I was trying to free up some of the machine's storage while I continued to use it.

**Comment by sr320** [comment]
I think you have separate issues re large files and repos. Lets discuss IRL.

**Comment by kubu4** [comment]
> Lets discuss IRL.

Good topic for lab meeting?

**Comment by yaaminiv** [comment]
> Good topic for lab meeting?

@kubu4 Just what I was thinking

**Comment by yaaminiv** [comment]
@sr320 suggested I move all of the large files I generate in my repo to gannet (they wouldn't make it on Github anyways, so why keep them in a repo). I successfully mounted gannet on hummingbird, but do not appear to have write-access in the spartina folder.

I looked at the [Data Management wiki](https://github.com/RobertsLab/resources/wiki/Data-Management) which suggests navigating to gannet.fish.washington.edu and uploading files. When I do that, I get this screen:

![screen shot 2018-10-10 at 1 03 30 pm](https://user-images.githubusercontent.com/22335838/46762783-edafa880-cc8c-11e8-924c-86df9c08d167.png)

TL;DR both methods of adding files to my gannet folder do not work.

**Comment by yaaminiv** [comment]
Edit: I think I just need my privileges updated. I was able to access File Station by going to https://128.95.149.52:5001

![untitled](https://user-images.githubusercontent.com/22335838/46762934-454e1400-cc8d-11e8-8ad2-d713c3af934b.png)

**Comment by sr320** [comment]
I updated index.html so wiki should be good

thanks
Steven


Steven B. Roberts
Kenneth K. Chew Endowed Professor
School of Aquatic and Fishery Sciences
University of Washington, Seattle WA
sr320@uw.edu - 206.866.5141
robertslab.info - @sr320
